src/template.lib.php

```diff
@@ -81,6 +81,22 @@ private static function replacePseudoVars($content = '', $params = array())
             }
         }
 
+        global $locale;
+
+        if ( $locale !== 'en_US' ) {
```

Comment:
When `$locale == 'en_US'` we're not removing the `{{` `}}`

Reply:
Oh thanks! Fixed in https://github.com/Sucuri/sucuri-wordpress-plugin/pull/72/commits/ab7bec3b77251e800e2ea9f6cb9aceaa2aa1b50b